lib/ethpy/ethpy/test_fixtures/local_chain.py

```diff
@@ -217,7 +222,7 @@ def launch_local_hyperdrive_pool(
         minimumTransactionAmount=minimum_transaction_amount.scaled_value,
         positionDuration=position_duration,
         checkpointDuration=checkpoint_duration,
-        timeStretch=time_stretch.scaled_value,
+        timeStretch=0,
```

Comment:
why is this zero now? if it's not used, shouldn't we delete it from the pool deploy config?

Reply:
This is what's in the migration script right now. This is required as it's a required field to the solidity call